### PR TITLE
Ratification of the Community Code of Conduct

### PR DESCRIPTION
**Pull request by ApolloZhu**
## Review Timeline

- [x] Jun 25 - July 10: @UWAppDev/board-member
- [x] July 11 - July 19: @ApolloZhu forgot to send out the anonymous feedback form
- [x] July 20 - July 30: All @UWAppDev members give feedback using [feedback form](https://docs.google.com/forms/d/e/1FAIpQLSfTqFMdf5fh9ayX0wDJ4I2J8Eti-d2Z21pEEXRen4gokt04qw/viewform?usp=sf_link)
- [x] July 30 - July 31: **`Current Stage ->`** Vote, all @UWAppDev/board-member should complete a pull request review
- [ ] Aug 1, 2021: In effect if passed (2/3 approvals)

## Suggest Changes

- Use [Suggest Changes](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request) if possible
- Otherwise, open a ***pull request*** against the [CODE_OF_CONDUCT.md](https://github.com/UWAppDev/community/blob/ratify-code-of-conduct/CODE_OF_CONDUCT.md) file in the [ratify-code-of-conduct](https://github.com/UWAppDev/community/tree/ratify-code-of-conduct) branch, so we can do a quick review on the amendments before merging to the original proposal.

## Note

- You should review the [entire Code of Conduct](https://github.com/UWAppDev/community/blob/ratify-code-of-conduct/CODE_OF_CONDUCT.md), not just the changes present in this pull request.
- It's possible to make revisions in the future/after the dedicated review periods.
- ***All* pull request review approvals will be dismissed whenever there’s a new change** to the doc, so you don't have to approve in the earlier stages.
- Two thirds out of 6 people is 4, pull request author cannot review their own pull request. Most likely the author supports whatever they are proposing?, so the required number of approving reviews is 3. Moving forward, the official club account @AppDevUW will be used to initiate pull requests so the original author can also submit a review.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,10 +1,8 @@
 # Code of Conduct
 
-> This is the ***TENTATIVE*** Code of Conduct of the Mobile Development Club at University of Washington.
-
 ## 1. Purpose
 
-A primary goal of Mobile Development Club is to be inclusive to the largest number of community members, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe and welcoming environment for all, regardless of 
+A primary goal of the ***Mobile Development Club** at University of Washington* is to be inclusive to the largest number of community members, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe and welcoming environment for all, regardless of
```
Comment (personalizedrefrigerator) [inline CODE_OF_CONDUCT.md:5]: Maybe *at **the** University of Washington*? Other than that, this looks good to me!

**Review by XinyiXiang — DISMISSED** [review]
approve

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (ApolloZhu) [reply CODE_OF_CONDUCT.md:5]:
```suggestion
A primary goal of the ***Mobile Development Club** at the University of Washington* is to be inclusive to the largest number of community members, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe and welcoming environment for all, regardless of 
```

**Review by zhinuoli — DISMISSED** [review]
Reviewed the entire Code of Conduct, and looks good!

**Comment by ApolloZhu** [comment]
Here's another reminder to propose amendments through comments/inline suggestions/pull requests before we go into voting.

- [ ] Before Saturday: you can propose amendments freely
- [ ] Saturday: please refrain from proposing amendments, and all @UWAppDev/board-member can start to approve this PR
   - You can still propose amendments
- [ ] Sunday: last day to approve this PR, 4 approvals required to merge.
   - Please try to leave comment/request changes if you decide to not approve.

**Review by personalizedrefrigerator — DISMISSED** [review]

**Comment by ApolloZhu** [comment]
So far we have received no feedback from the community, I think we can assume there are no major changes required.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -17,7 +15,7 @@ A supplemental goal of this Code of Conduct is to increase open [source/culture/
 
 Communities mirror the societies in which they exist and positive action is essential to counteract the many forms of inequality and abuses of power that exist in society.
```
Comment (Lucas-ZX-W) [inline CODE_OF_CONDUCT.md:17]: To provide equal access of information to all, the Mobile Development Club shall refrain from actions and methods of broadcasts that unnecessarily exclude certain demographics or individuals from accessing information released and announced by the RSO (examples against this clause include but are not limited t0 making announcements solely to major and department-specific communication mailing-lists and other communication groups OR to explicitly state or conduct outreach in a fashion that suggests that the RSO is "exclusive to people in computing").

**Review by SilliJelli — DISMISSED** [review]

**Review by XinyiXiang — DISMISSED** [review]
neat

**Review by Lucas-ZX-W — CHANGES_REQUESTED** [review]
Proposed amendments to the COD from Lucas.

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -17,7 +15,7 @@ A supplemental goal of this Code of Conduct is to increase open [source/culture/
...
 
-If you see someone who is making an extra effort to ensure our community is welcoming, friendly, and encourages all participants to contribute to the fullest extent, we want to know.
+If you see someone who is making extra efforts to ensure our community is welcoming, friendly, and encourages all participants to contribute to the fullest extent, please let us know as well.
 
 ## 3. Expected Behavior
```
[the remark this answers is not in the document]
Comment (ApolloZhu) [reply CODE_OF_CONDUCT.md:21]: > Proposed amendments to the COD from Lucas.

Can you use a pull request or "suggest changes" feature so it's more clear what are the exact changes?

**Comment by ApolloZhu** [comment]
> Proposed amendments to the COD from Lucas.

Can you use a pull request or "suggest changes" feature so it's more clear what are the exact changes? @Lucas-ZX-W

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -17,7 +15,7 @@ A supplemental goal of this Code of Conduct is to increase open [source/culture/
 
 Communities mirror the societies in which they exist and positive action is essential to counteract the many forms of inequality and abuses of power that exist in society.
 
-If you see someone who is making an extra effort to ensure our community is welcoming, friendly, and encourages all participants to contribute to the fullest extent, we want to know.
+If you see someone who is making extra efforts to ensure our community is welcoming, friendly, and encourages all participants to contribute to the fullest extent, please let us know as well.
```
Comment (Lucas-ZX-W) [inline CODE_OF_CONDUCT.md:19]: **Addition:**
```suggestion
To ensure that all are welcomed at the Mobile Development Club, the RSO shall maintain an open, free, and neutral stance on the freedom of expression. RSO meetings and gatherings are expected to follow the rough topics of discussion as outlined by the agenda and the material presented. However, no **forceful** attempts shall be made to restrict the will to express of any individual or entity present. Non-forceful reactions to any discharge of expression, including but not limited to frowning at the expresser and expressing general disapproval are not entirely constructive to a welcoming atmosphere, but nonetheless shall also **NOT** be forcefully restricted to uphold the freedom of expression of all.
```

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (Lucas-ZX-W) [inline CODE_OF_CONDUCT.md:17]: **Addition:**
```suggestion
To provide equal access of information to all, the Mobile Development Club shall refrain from actions and methods of broadcasts that unnecessarily exclude certain demographics or individuals from accessing information released and announced by the RSO (examples against this clause include but are not limited to making announcements solely to major and department-specific communication mailing-lists and other communication groups OR to explicitly state or conduct outreach in a fashion that suggests that the RSO is "exclusive to people in computing").
```

File: CODE_OF_CONDUCT.md (hunk 3 above)
Comment (Lucas-ZX-W) [inline CODE_OF_CONDUCT.md:21]: **Change to existing:**
Note: GitHub's not letting me put a comment inside the collapsible section below, so this suggestion would swap out lines #52-54.

```suggestion
## 5. Armed Aggression and Hostility with Weapons Policy
To maintain an environment with minimal armed aggression and hostility, no weapons or armaments will be allowed at Mobile Development Club events, community spaces, or in other spaces covered by the scope of this Code of Conduct. Weapons and armaments include but are not limited to projectile weapons and armaments (firearms, pneumatic arms, bows, and crossbows, etc.) explosives (including fireworks), and large knives such as those used for hunting or display, as well as any other item used for the purpose of causing injury or harm to others. Anyone seen in possession of one of these items will be asked to leave immediately, and will only be allowed to return unarmed without any weapon. Community members are further expected to comply with all state and local laws on this matter.
```

**Review by Lucas-ZX-W — APPROVED** [review]

**Review by SilliJelli — APPROVED** [review]

**Review by zhinuoli — APPROVED** [review]

**Review by personalizedrefrigerator — APPROVED** [review]

**Pull request merged**